packages/shared/src/components/Icon.tsx

```diff
@@ -27,11 +27,9 @@ export const iconSizeToClassName: Record<IconSize, string> = {
 
 type ItemType = React.ComponentType<{ className }>;
 
-export interface IconProps {
+export interface IconProps extends ComponentProps<'svg'> {
   secondary?: boolean;
   size?: IconSize;
```

Comment:
Need to make the icons able to receive `aria-` attributes

Reply:
ComponentProps is prefered anyway :)